### PR TITLE
maintainers should be maintainers

### LINKED ISSUE #49: Repo configuration: repo-approvers team maintainers
I think `repo-maintainers` team members should be set as maintainers of `repo-approvers` team. Need to document it

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Fixes #49 #84

### CODE REVIEW

File: docs/how-to-configure-new-repository.md [hunk 1]
```diff
@@ -14,18 +14,24 @@ Documents [Community Membership](../community-membership.md) and
...
-   named `foo-approvers` and `foo-maintainers`. `foo-maintainers` is a child of
-   `foo-approvers` as it always contains subset of people and defines larger
-   scope of privileges.
-2. Both teams has `Write` permissions for the repository.
-3. Root-level `CODEOWNERS` file on the repository should include superset of
+1. Every repository has two teams associated with it. Typically for the
+   repository `opentelemetry-foo` they will be named `foo-approvers` and
+   `foo-maintainers`. `foo-maintainers` is a child of `foo-approvers` as it
+   always contains subset of people and defines larger scope of privileges.
+2. Event though the members of `foo-maintainers` are the members of
```
Comment (bhs) [inline docs/how-to-configure-new-repository.md]: "Even though"

File: docs/how-to-configure-new-repository.md [hunk 2]
```diff
@@ -14,18 +14,24 @@ Documents [Community Membership](../community-membership.md) and
...
+   `foo-approvers` automatically as the groups are enclosed, every member of
+   `foo-maintainers` should be included in `foo-approvers` explicitly with the
+   "Maintainer". So repository maintainers can invite new approvers.
```
Comment (bhs) [inline docs/how-to-configure-new-repository.md]: what is `the "Maintainer"`?

Comment (bhs) [reply docs/how-to-configure-new-repository.md]: you mean `the "Maintain" GitHub permissions`???

Comment (bogdandrutu) [reply docs/how-to-configure-new-repository.md]: yes, the new Maintain github permissions, I guess.

Comment (SergeyKanzhelev) [reply docs/how-to-configure-new-repository.md]: I mean team maintainer role. So maintainers will be able to add approvers

image

File: docs/how-to-configure-new-repository.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply docs/how-to-configure-new-repository.md]:
```suggestion
2. Even though the members of `foo-maintainers` are the members of
```

File: docs/how-to-configure-new-repository.md (hunk 2 above)
Comment (bhs) [reply docs/how-to-configure-new-repository.md]: Gotcha. Let's say

```
Even though the members of `foo-maintainers` are included in `foo-approvers` transitively, every member of `foo-maintainers` should be included in `foo-approvers` explicitly, and with the "Maintainer" GitHub privileges.
```

Comment (SergeyKanzhelev) [reply docs/how-to-configure-new-repository.md]: fixed

**Pull request merged**